source/Xunit.Gherkin.Quick.ProjectConsumer/GivenWhenThenTests/FeatureWithBackground.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: With Background
+	In order to use common steps are the start of each scenario
```

Comment:
I think there is a typo. Must be "at the start" I think, but it says "are the start".